src/Uno.UI/UI/Xaml/Controls/Slider/Slider.mux.cs

```diff
@@ -431,8 +431,12 @@ private void OnPointerPressed(object sender, PointerRoutedEventArgs args)
 	}
 
 	// PointerMoved event handler.
-	private void OnPointerMoved(object sender, PointerRoutedEventArgs args)
+	private void OnPointerMoved(object sender, PointerRoutedEventArgs args) => OnPointerMoved(args);
```

Comment:
It would be best to understand why this change is needed, because this whole code was ported from WinUI, so it means that discrepancy in behavior should mean that something around pointer handling is not properly aligned in our case